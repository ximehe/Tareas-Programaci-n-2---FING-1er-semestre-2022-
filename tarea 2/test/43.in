# implementar copiaCadena
copiaCadena
insertarAlInicio 5 5.0
copiaCadena
insertarAlInicio 6 6.0
copiaCadena
cadenaSiguiente 1
copiaCadena
insertarAlInicio 9 6.0
insertarAlInicio 8 8.0
insertarAlInicio 7 7.0
insertarAlInicio 4 4.0
insertarAlInicio 3 3.0
cadenaSiguiente 1
copiaCadena
Fin