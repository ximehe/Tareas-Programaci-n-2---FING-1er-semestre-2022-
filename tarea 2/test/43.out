1>#  implementar copiaCadena.
2>
3>Se insertó (5,5.00) al inicio de la cadena.
4>(5,5.00)
5>Se insertó (6,6.00) al inicio de la cadena.
6>(6,6.00)(5,5.00)
7>(5,5.00)(6,6.00)
8>(6,6.00)(5,5.00)
9>Se insertó (9,6.00) al inicio de la cadena.
10>Se insertó (8,8.00) al inicio de la cadena.
11>Se insertó (7,7.00) al inicio de la cadena.
12>Se insertó (4,4.00) al inicio de la cadena.
13>Se insertó (3,3.00) al inicio de la cadena.
14>(4,4.00)(7,7.00)(8,8.00)(9,6.00)(6,6.00)(5,5.00)(3,3.00)
15>(3,3.00)(4,4.00)(7,7.00)(8,8.00)(9,6.00)(6,6.00)(5,5.00)
16>Fin.
